solidago/README.md

```diff
@@ -0,0 +1,3 @@
+# Solidago
+
+Algorithms for the Tournesol platform
```

Comment:
It could be nice to introduce the package with its short description. If I remember correctly:

**So**lid **Al**gorithm **Go**vernance